incubator/command-generate/src/main/java/io/aklivity/zilla/runtime/command/generate/internal/asyncapi/view/SchemaView.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.command.generate.internal.asyncapi.view;
+
+import java.util.List;
+import java.util.Map;
+
+import jakarta.json.bind.annotation.JsonbPropertyOrder;
+
+import io.aklivity.zilla.runtime.command.generate.internal.asyncapi.model.Item;
+import io.aklivity.zilla.runtime.command.generate.internal.asyncapi.model.Schema;
+
+@JsonbPropertyOrder({
+    "type",
+    "items",
+    "properties",
+    "required"
+})
+public final class SchemaView extends Resolvable<Schema>
+{
+    private static final String ARRAY_TYPE = "array";
+
+    private final Schema schema;
+    private final Map<String, Schema> schemas;
+
+    private SchemaView(
+        Map<String, Schema> schemas,
+        Schema schema)
+    {
+        super(schemas, "#/components/schemas/(\\w+)");
+        if (schema.ref != null)
+        {
+            schema = resolveRef(schema.ref);
+        }
+        else if (ARRAY_TYPE.equals(schema.type) && schema.items != null && schema.items.ref != null)
+        {
+            schema.items = resolveRef(schema.items.ref);
+        }
+        this.schemas = schemas;
+        this.schema = schema;
+    }
+
+    public String getType()
```

Comment:
I am using `jsonb` for serialization. I didn't find a workaround to change the method names from `get...()`.  The `@JsonbProperty` annotation seems to work only on methods that follow the naming convention.